ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -636,7 +649,7 @@ data TraceAddBlockEvent blk =
     -- chain (second fragment).
   | AddedToCurrentChain
       [LedgerEvent blk]
-      (NewTipInfo blk)
+      (SelectionChangedInfo blk)
```

Comment:
Once this diff context appear it occurred to me that perhaps the use cases could just get the `SelectView` from the two header chain arguments.

However, Esgen pointed out the call site would then need to also have the `BlockConfig` in scope. (That's certainly always _possible_, but likely often onerous).